hack/ostree_tag.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+declare -a tags
 if pkg-config ostree-1 2> /dev/null ; then
-	echo containers_image_ostree
+	tags+=(containers_image_ostree)
 fi
+
+if test $(${GO:-go} env GOOS) != "linux" ; then
+	exit 0
+fi
+cc -E - $(pkg-config --cflags ostree-1) > /dev/null 2> /dev/null << EOF
+#include <ostree-1/ostree.h>
+EOF
+if test $? -eq 0 ; then
+	tags+=(ostree)
+fi
+
+echo "${tags[@]}"
```

Comment:
(This test seems to come from c/storage.)

Is there a reason for the `ostree` and `containers_image_ostree` logic to differ? Both seem to have the same dependencies (CGo builds using `libostree-1`), so it seems surprising to make the decision logic different.

The new code is more comprehensive, so I have no objections to using that one, but having two different paths in the check for an unknown reason just seems surprising.

Reply:
Actually I just copied here the ostree_tag.sh script from c/storage, as you realized.
Honestly I did not think about the differences, thank you for pointing it out.

(On openSUSE) I see that `pkg-config` succeeds if and only if the `libostree-devel` package is installed (the one including C headers). Because of this, checking the result of `pkg-config ostree-1` seems to be more strict than `cc -E`, that's only checking the presence of the headers files, as you also pointed out.

So, in the end, a single check on `pkg-config ostree-1` is adequate for both tags.

I'm updating the PR, thanks!